synapse/storage/databases/main/events_worker.py

```diff
@@ -62,7 +62,8 @@
 EVENT_QUEUE_THREADS = 3  # Max number of threads that will fetch events
```

Comment:
Seeing a lot of tests fail due to timeouts over replication. I am not having much luck understanding why, and I'm not able to reproduce the problem in a live environment.